tests/lax_control_flow_test.py

```diff
@@ -1951,6 +1951,21 @@ def testScanBodyCarryTypeMismatchErrors(self):
                                   x[2]), None),
                    (jnp.array(0, 'int32'),) * 3, None, length=1)
 
+  def testScanInvalidUnrollRaises(self):
+    # 'jax_enable_checks' must be set to `False` to trigger this error because
+    # otherwise the invalid value for unroll is caught by `_scan_typecheck`.
+    prev = jax.config.jax_enable_checks
```

Comment:
You can use `with jax.enable_checks(False)` to do this automatically; or alternatively decorate the test method with `@jax.enable_checks(False)`

Reply:
Thanks! I couldn't find that in an initial search. Updated!